src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,30 @@
+package calculator
+
+class Calculator {
+    fun calculate(input: String): Int {
+        if (input.isBlank()) {
+            throw IllegalArgumentException("입력값이 공백입니다.")
```

Comment:
수식이 공백인 경우에 대해서 예외처리 잘 해주셨네요! 🤩👍

Kotlin에서 제공하는 [require](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html), [check](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/check.html) 함수를 활용해서 아래와 같이 더 간단하게 변경해볼 수 있을 것 같아요. 😃
```suggestion
            require(input.isNotBlank()) { "입력값이 공백입니다." }
```

Reply:
더 깔끔해졌네요 유용한 구문 감사합니다 😀